lib/ansible/modules/net_tools/ip_link_device_attribute.py

```diff
@@ -0,0 +1,548 @@
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ip_link_device_attribute
+version_added: "2.10"
+author: "George Shuklin (@amarao)"
+short_description: Set link-level properties for network interfaces for Linux
+requirements: [iproute2]
+description: >
+    Allows to query or change link-level attrubutes of interfaces such as
+    administrative state (up/down), MTU, arp, multicast, promisc, txqueuelen,
+    address, broadcast, netns, alias.
+    It currently does not support some features of ip link set, as
+    allmulticast, dynamic, protodown, trailers, link-netnsid, xdp, xdpgeneric,
+    xdpdrv, xdpoffload, master/nomaster, addrgenmode, macaddr, vrf.
+
+options:
+    name:
+        type: str
+        required: false
+        aliases: [device]
+        description:
+            - Name of the network device.
+            - Either I(name) or I(group_id) should be given.
+
+    group_id:
+        type: str
+        required: false
+        description:
+            - Id of group of interfaces.
+            - Either I(name) or I(group_id) should be given.
+            - Settings are applied for all interfaces in the group.
+
+    namespace:
+        type: str
+        required: false
+        description:
+            - Name of namespace where interface is present.
+            - Do not mix up with netns option
+              (which allows to change network namespace).
+
+    state:
+        type: str
+        choices: [up, down]
+        required: false
+        description:
+            - controls the state of interface. Up or down.
+
+    arp:
+        type: bool
+        required: false
+        description:
+            - Enable or disable ARP on an interface
+
+    multicast:
+        type: bool
+        required: false
+        description:
+            - Set on or off MULTICAST flag on device
+
+    promisc:
+        type: bool
+        required: false
+        description:
+            - Enable or disable promiscuous mode for interface
+
+    txqueuelen:
+        type: int
+        required: false
+        description:
+            - Set transmit queue length of the interface
+            - Default value for kernel is usually 1000
+
+    mtu:
+        type: int
+        required: false
+        description:
+            - Set MTU value for interface.
+            - Default value is defined by the kernel and it is 1500
+            - This is L3 mtu (limit on size of IP packets)
+
+    address:
+        type: str
+        required: false
+        description:
+            - MAC address (L2 address) for the interface
+            - Some interfaces can not have L2 address. Module will fail
+              if address is specified for interface without L2 address
+              support (f.e. 'type vcan')
```

Comment:
```suggestion
              support (for example 'type vcan').
```

f.e. isn't generally used.  I'd either use 'for example' or 'e.g.'  ( if you listed the only examples you could also use 'i.e.')